extension/src/background/messageListener/popupMessageListener.ts

```diff
@@ -910,35 +921,19 @@ export const popupMessageListener = (request: Request) => {
         JSON.stringify(currentNetworksList),
       );
       localStorage.setItem(NETWORK_ID, JSON.stringify(defaultNetworkDetails));
+      await dataStorage.setItem({
+        [NETWORKS_LIST_ID]: currentNetworksList,
+        [NETWORK_ID]: defaultNetworkDetails,
+      });
     }
 
     localStorage.setItem(
       IS_EXPERIMENTAL_MODE_ID,
       JSON.stringify(isExperimentalModeEnabled),
     );
-
-    if (isExperimentalModeEnabled !== currentIsExperimentalModeEnabled) {
```

Comment:
As a result of a bade merge, this block was in here twice, so removing this 🤦

Reply:
😬  oof nice catch